--- xephem-3.7.7/GUI/xephem/Makefile.orig	2015-08-09 23:36:50.000000000 +0200
+++ xephem-3.7.7/GUI/xephem/Makefile	2018-12-19 21:10:25.456956449 +0100
@@ -10,8 +10,8 @@
 # one executable, xephem.
 
 # These -I and -L flags point to the supporting XEphem libraries
-LIBINC = -I../../libastro -I../../libip -I../../liblilxml -I../../libjpegd -I../../libpng -I../../libz
-LIBLNK = -L../../libastro -L../../libip -L../../liblilxml -L../../libjpegd -L../../libpng -L../../libz
+LIBINC = -I../../libastro -I../../libip -I../../liblilxml
+LIBLNK = -L../../libastro -L../../libip -L../../liblilxml
-LIBLIB = -lastro -lip -llilxml -ljpegd -lpng -lz
+LIBLIB = -lastro -lip -llilxml -ljpeg -lpng -lz
 
 # MOTIFI is the directory containing the Xm directory of include files.
@@ -32,7 +32,7 @@
 # for linux and Apple OS X
 CC = gcc
 CLDFLAGS = -g
-CFLAGS = $(LIBINC) $(CLDFLAGS) -O2 -Wall -I$(MOTIFI) -I/opt/X11/include
+CFLAGS = $(LIBINC) $(CLDFLAGS) -Wall -I$(MOTIFI) -I/opt/X11/include
 LDFLAGS = $(LIBLNK) $(CLDFLAGS) -L$(MOTIFL) -L/opt/X11/lib
 XLIBS = -lXm -lXp -lXt -lXext -lXmu -lX11
 LIBS = $(XLIBS) $(LIBLIB) -lm
@@ -199,8 +199,5 @@
 	cd ../../libip; make
-	cd ../../libjpegd; make
 	cd ../../liblilxml; make
-	cd ../../libpng; make
-	cd ../../libz; make
 
 clean:
 	rm -fr *.o ../../lib*/*.[ao]
